packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1784,6 +1784,17 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
 
   Offset? _lastTapDownPosition;
 
+  /// The position of the most recent tap down event on this text input.
+  Offset? get lastTapDownPosition => _lastTapDownPosition;
```

Comment:
I had to expose this in order to position the menu at the tap position (on mobile, it's positioned centered on the selection).  Any objections to this?

Reply:
Probably OK in the short term. We should really be using the existing menu system to show desktop menus.